Adds UI for API docs import

helrond opened the pull request:
Adds a page with a form to upload Fluxx API docs, plus messages which indicate success or failure. The logic behind this form parses the table name from the document itself, so users do not need to name the document in any specific way.

Questions for review:
- Is a more proper pattern for a failed import to redirect back to the import form page (rather than the home page)
- Are the instructions in the README clear and still in the correct overall order?

fixes #95

HaSistrunk commented:
Nice! This is an awesome enhancement and works well for me. I think there might be opportunities for help text and language alignment in the README, but we might have a better idea of what that looks like after we address issue #96 .

> Is a more proper pattern for a failed import to redirect back to the import form page (rather than the home page)

Yes, I think it's best practice for cognitive load to keep error indicators close to the interface where the error occurred.

> Are the instructions in the README clear and still in the correct overall order?

It's a little confusing in the ##Configuration section that it tells the user to navigate to /admin, but then step 5 is the tables and fields configuration (which I know can be done in admin, but we aren't recommending that). But once you get into the specific instructions that it links to, that becomes more clear. 

A couple ideas:
- Add a link to the /import page in the README. 
- Clarify that these need to be HTML files, right? That could also be added to the form labels: "Grant request HTML file" OR help text "Select API documentation HTML file/s that you downloaded from Fluxx".

helrond commented:
Cool, thanks! Did a couple of things here:
- Updated the view logic so users will stay on the import form page if an error occurs during import
- Updated the form labels to indicate that the uploaded files should be HTML files
- Linked to the import page from the README
